arm_robots/src/arm_robots/panda.py

```diff
@@ -11,81 +16,52 @@
                                     **kwargs)
-        self.nebula_arm = "nebula_arm"
-        self.nebula_id = "panda_1"
-        self.rocket_arm = "rocket_arm"
-        self.rocket_id = "panda_2"
-        self.nebula_wrist = "panda_1_link8"
-        self.rocket_wrist = "panda_2_link8"
-        self.nebula_gripper = PandaGripper(self.robot_namespace, self.nebula_id)
-        self.rocket_gripper = PandaGripper(self.robot_namespace, self.rocket_id)
-        rospy.Subscriber(ns_join(self.robot_namespace, f'{self.nebula_id}_state_controller/franka_states'), FrankaState,
-                         self.nebula_error_cb)
-        rospy.Subscriber(ns_join(self.robot_namespace, f'{self.rocket_id}_state_controller/franka_states'), FrankaState,
-                         self.rocket_error_cb)
-        self.error_client = actionlib.SimpleActionClient(ns_join(self.robot_namespace, "error_recovery"),
-                                                         ErrorRecoveryAction)
-        self.error_client.wait_for_server()
-
-    def open_nebula_gripper(self):
-        self.nebula_gripper.move(self.nebula_gripper.MAX_WIDTH)
-
-    def close_nebula_gripper(self):
-        self.nebula_gripper.move(self.nebula_gripper.MIN_WIDTH)
-
-    def open_rocket_gripper(self):
-        self.rocket_gripper.move(self.rocket_gripper.MAX_WIDTH)
-
-    def close_rocket_gripper(self):
-        self.rocket_gripper.move(self.rocket_gripper.MIN_WIDTH)
-
-    def nebula_error_cb(self, data):
-        if data.current_errors.communication_constraints_violation or data.last_motion_errors.communication_constraints_violation:
-            self.clear_errors(wait_for_result=True)
-
-    def rocket_error_cb(self, data):
-        if data.current_errors.communication_constraints_violation or data.last_motion_errors.communication_constraints_violation:
-            self.clear_errors(wait_for_result=True)
-
-    def clear_errors(self, wait_for_result=False):
-        goal = ErrorRecoveryGoal()
-        self.error_client.send_goal(goal)
-        if wait_for_result:
-            result = self.error_client.wait_for_result(rospy.Duration(10))
-            return result
-        return True
-
+        self.panda_1 = 'panda_1'
+        self.panda_2 = 'panda_2'
+        self.display_goals = False
+        # self.panda_1.gripper = PandaGripper(self.robot_namespace, self.panda_1)
+        # self.panda_2.gripper = PandaGripper(self.robot_namespace, self.panda_2)
+
+    def follow_arms_joint_trajectory(self, trajectory: JointTrajectory, stop_condition: Optional[Callable] = None,
+                                     group_name: Optional[str] = None):
+        move_group = self.get_move_group_commander(group_name)
+        plan_msg = RobotTrajectory()
+        plan_msg.joint_trajectory = trajectory
+        move_group.execute(plan_msg)
+        pass
+
+    def send_joint_command(self, joint_names: List[str], trajectory_point: JointTrajectoryPoint) -> Tuple[bool, str]:
+        pass
```

Comment:
this should probably `raise NotImplementedError` to avoid people accidentally calling this somehow